x/farming/client/cli/utils.go

```diff
@@ -27,6 +27,7 @@ type PrivateRatioPlanRequest struct {
 	StartTime          time.Time    `json:"start_time"`
 	EndTime            time.Time    `json:"end_time"`
 	EpochRatio         sdk.Dec      `json:"epoch_ratio"`
+	RewardDenoms       []string     `json:"reward_denoms"`
```

Comment:
Can you add this param in `TestParsePrivateRatioPlan` case?

Reply:
Nice catch. I added that check.